sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -313,22 +312,22 @@ def get_queue_properties(self, **kwargs):
         except HttpResponseError as error:
             process_storage_error(error)
         response.name = self.queue_name
-        return response # type: ignore
+        return response
 
     @distributed_trace
-    def set_queue_metadata(self,
-                           metadata=None,  # type: Optional[Dict[str, Any]]
-                           **kwargs  # type: Any
-                           ):
-        # type: (...) -> None
+    def set_queue_metadata(
+            self,
+            metadata: Optional[Dict[str, Any]] = None,
+            **kwargs: Any
+        ) -> None:
         """Sets user-defined metadata on the specified queue.
 
         Metadata is associated with the queue as name-value pairs.
 
         :param metadata:
             A dict containing name-value pairs to associate with the
             queue as metadata.
-        :type metadata: dict(str, str)
```

Comment:
But interestingly enough the docstring says `dict(str, str)` so curious where the conflicting typehint information came from